ansible/roles/uan_interfaces/tasks/main.yml

```diff
@@ -212,7 +217,15 @@
 
   # Configure CAN v2 - Uses external subnet on vlan007
   - include_tasks: can-v2.yml
-    when: uan_can_setup
+    when: uan_can_setup and not uan_chn_setup
+
+  # Configure CHN - Uses HSN for customer user access
+  - include_tasks: chn.yml
+    when: uan_chn_setup and not uan_can_setup
+
+  # Configure CHN - Uses HSN for customer user access
+  - include_tasks: chn.yml
+    when: uan_chn_setup and not uan_can_setup
```

Comment:
A couple of comments here:

* It looks like chn.yml is included twice
* Using `when` on both variables seems unnecessary when you've already checked only one can be true 
* I'm wondering if this could be handled with a single variable like "uan_customer_access" and have that be set to either CAN or CHN. The benefit to this approach would be a simplification of `can-v2.yml` and `chn.yml` into a single `customer-access.yml`. There's a fair amount of duplication between `can-v2.yml` and `chn.yml` (get the SLS networks, get the customer access ips, wicked restarts, etc...) and that could be eliminated if we consolidate the code paths. At the very least, the common set of tasks should be deduplicated where possible.

Note that this could lead to some backwards incompatibility if we don't map `uan_can_setup: true` to `uan_customer_access: can` and enforce some checking logic.